play/wikirecent/app/utils/async_tail.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""async_tail
+
+Module showing an example for how to asynchronously TAIL a Materialized view.
```

Comment:
Maybe clarify that this doesn't use the python webserver? "Example for how to asynchronously TAIL a Materialized view in python with no intermediary server"

Reply:
Will do!